crates/ruff/src/rules/flake8_variables_names/settings.rs

```diff
@@ -0,0 +1,44 @@
+//! Settings for the `flake8-variables-names` plugin.
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use ruff_macros::{CacheKey, ConfigurationOptions};
+
+#[derive(
+    Debug, PartialEq, Eq, Serialize, Deserialize, Default, ConfigurationOptions, JsonSchema,
+)]
+#[serde(
+    deny_unknown_fields,
+    rename_all = "kebab-case",
+    rename = "Flake8VariablesNamesOptions"
+)]
+pub struct Options {
+    #[option(
+        default = "false",
+        value_type = "bool",
+        example = "use-varnames-strict-mode = true"
+    )]
+    /// Whether to use strict settings for the allow/block lists
```

Comment:
Can we add some notes on the effect that strict `True` vs. `False` has on the checks?

Reply:
definitely; it has been updated